api/src/main/java/me/lokka30/treasury/api/economy/account/SharedAccountPermission.java

```diff
@@ -11,42 +11,42 @@
 import me.lokka30.treasury.api.economy.transaction.EconomyTransactionInitiator;
 
 /**
- * Enum that holds the permissions of a {@link BankAccount}.
+ * Enum that holds the permissions of an {@link Account} that is shared among multiple players.
  *
  * @author MrNemo64
- * @see BankAccount
+ * @see Account
  * @since {@link me.lokka30.treasury.api.economy.misc.EconomyAPIVersion#v1_0 v1.0}
  */
-public enum BankAccountPermission {
+public enum SharedAccountPermission {
```

Comment:
By calling this `SharedAccountPermission` and being used by all bank accounts we are saying that PlayerAccounts are also shared accounts, whitch may or may not be true, it is left for the provider to specify.

Reply:
I think this naming scheme better suits it since the methods are in Account now.